### ISSUE TITLE
`operator_usage_whitespace` false positive with multiline generics

### ISSUE THREAD

**Issue by keith** [realm/SwiftLint#3381]
```
let foo = SomeGeneric<FirstType,
                      SecondType>()
```

With this code, you get a false positive that there should be whitespace around the `<>` operators:

```
foo.swift:1:22: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
foo.swift:2:33: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
```

**Comment by marcelofabri** [comment]
Duplicate of #3102
